api/config/cdr_versions_prod.json

```diff
@@ -10,6 +10,6 @@
     "releaseNumber": 1,
     "numParticipants": 946237,
     "cdrDbName": "cdr20181023",
-    "publicDbName": "public20181015"
+    "publicDbName": "public20181026"
```

Comment:
This doesn't exist in prod yet. Are these changes breaking, or can prod stay on the old one for now?

Reply:
I guess i would leave it as it is for now since databrowser is not functional in prod anyway.